src/libsync/propagateupload.h

```diff
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) by Olivier Goffart <ogoffart@owncloud.com>
+ * Copyright (C) by Piotr Mrowczynski <piotr@owncloud.com>
```

Comment:
Oh, on server side it makes sense for code maintenance reasons (you know whom to contact for details), can change if it this is different on the client.